pytket/extensions/cutensornet/tnstate/ttn.py

```diff
@@ -133,11 +133,6 @@ def __init__(
         self.nodes: dict[RootPath, TreeNode] = dict()
         self.qubit_position: dict[Qubit, tuple[RootPath, int]] = dict()
 
-        if self._cfg.truncation_fidelity < 1:
-            raise NotImplementedError(

```

Comment:
It is still listed in the docstrings tho :)

Reply:
Good catch!